src/lib.rs

```diff
@@ -227,8 +298,10 @@ impl Picture {
 
 impl Drop for Picture {
     fn drop(&mut self) {
-        unsafe {
-            dav1d_picture_unref(Arc::get_mut(&mut self.pic).unwrap());
+        if let Some(pic) = Arc::get_mut(&mut self.pic).take() {
```

Comment:
While this probably works, this is more a hack. It would be better to implement `Drop` on the inner part of the `Arc` instead, i.e. have a smaller wrapper struct around the `Dav1dPicture` that implements `Drop`.